README.md

```diff
@@ -97,6 +97,7 @@ These commands can only be performed by users with the role of 'ADMIN' or users
 * `@PockyBot reset` &mdash; remove all pegs from the database
 * `@PockyBot update` &mdash; update names stored in the database with users' current display names
 * `@PockyBot finish` &mdash; complete the cycle by displaying the winners and results before wiping the database
+* `@PockyBot get|set|refresh|delete {name|userid} {value}` &mdash; get or change config values
```

Comment:
Why `{userid}`? Why not `@User` and get the userid from that?

Reply:
Yeah I know, that's a change I think needs to happen but I was just fixing things up as they are now